package.json

```diff
@@ -54,6 +54,7 @@
     "bluebird": "^3.5.0",
     "browser-request": "^0.3.3",
     "content-type": "^1.0.2",
+    "olm": "https://matrix.org/packages/npm/olm/olm-2.2.2.tgz",
```

Comment:
olm was deliberately omitted here for some reason involving making it possible to do without it. the details currently escape me, but what is the reason for adding it?

Reply:
bah, because npm now adds any package you install to package.json even if you don't specify --save.